docs/cli-reference.md

```diff
@@ -607,18 +607,25 @@ The URLs that are generated using this command, can be used to access the deploy
   source <(odo utils terminal zsh)
 
 
-   # Set a configuration value
-   odo utils config set UpdateNotification false
-   odo utils config set NamePrefix "app"
-   odo utils config set Timeout 20
+   # Set a global configuration value
+   odo utils config set --global UpdateNotification false
+   odo utils config set --global NamePrefix "app"
+   odo utils config set --global timeout 20
```

Comment:
Why timeout starts with lowercase, but everything else with upper?

Reply:
ack

Reply:
need to generate the cli-reference

Reply:
resolved